library/src/components/supportedExtensions/XExtension.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { ExtensionComponentProps } from '../../types';
+
+/**
+ * See <https://github.com/asyncapi/extensions-catalog/blob/master/extensions/x.md>.
+ */
+export default function XExtension({
+  propertyValue,
+}: ExtensionComponentProps<string>) {
+  const onClickHandler = () => {
+    window.open(`https://x.com/${propertyValue}`, '_blank');
```

Comment:
I think for best practices we should include the `noopener,noreferrer` attribute

Reply:
I added the attribute in the anchor tag.